presto-main/src/main/java/com/facebook/presto/operator/TableWriterOperator.java

```diff
@@ -78,6 +78,7 @@
 public class TableWriterOperator
         implements Operator
 {
+    public static final String OPERATOR_TYPE_NAME = "TableWriterOperator";
```

Comment:
Why this one is OPERATOR_TYPE_NAME and other operators are OPERATOR_TYPE? Please unify the names.

Reply:
@yingsu00 made the required changes. Can you please verify